test/test-jsdomparser.js

```diff
@@ -6,7 +6,7 @@ var readability = require("../index.js");
 var JSDOMParser = readability.JSDOMParser;
 
 var BASETESTCASE = '<html><body><p>Some text and <a class="someclass" href="#">a link</a></p>' +
-                   '<div id="foo">With a <script>With < fancy " characters in it because' +
+                   '<div id="foo">With a <script>With &lt; fancy " characters in it because' +
```

Comment:
I think that if it works without the replacement we should leave the files alone.

I think replacing the `<` and `>` that start/end CDATA sections (like my comment below) is definitely wrong.